src/js/components/chat-input.jsx

```diff
@@ -17,11 +17,16 @@ export class ChatInputComponent extends Component {
     }
 
     onChange(e) {
+        checkAndResetUnreadCount();
```

Comment:
Should we throttle this maybe? `store.getState().conversation.unreadCount` is only reset to `0` after the http call completes - if someone types a lot of characters we're gonna spawn a whole ton of POST requests